components/core/src/clp/streaming_archive/Constants.hpp

```diff
@@ -15,6 +14,16 @@ constexpr char cMetadataFileName[] = "metadata";
 constexpr char cMetadataDBFileName[] = "metadata.db";
 constexpr char cSchemaFileName[] = "schema.txt";
 
+namespace cArchiveFormatVersion {
+constexpr uint8_t cDefaultVersionMajor{0};
+constexpr uint8_t cDefaultVersionMinor{1};
+constexpr uint16_t cDefaultVersionPatch{0};
+constexpr archive_format_version_t cDefaultVersion{
```

Comment:
If we keep these inside the namespace, we should remove the prefix `c` to follow the convention of the other constants in this file (again, this is an exception to our clang-tidy rules).